internal/services/jsonrpc/usersvc/service.go

```diff
@@ -519,6 +551,7 @@ func (svc *Service) Call(ctx context.Context, req *userjson.CallRequest) (*userj
 
 	return &userjson.CallResponse{
 		Result: btsResult,
+		Logs:   logs,
```

Comment:
I'm not seeing any concurrency controls with the closure goroutine above that appends to this.

Reply:
Ah yeah I guess we should close before this.